fetch.bs

```diff
@@ -2349,38 +2348,59 @@ steps:
    <a>connection</a>.
   </ol>
 
+ <li>
+  <p>Let <var>proxies</var> be the result of performing proxy resolution for <var>url</var> in an
+  <a>implementation-defined</a> manner. If a proxy preference order has not been explicitly
+  configured, then SOCKS proxies should be preferred to HTTPS proxies which should be preferred to
+  HTTP proxies. If there is no proxy, let <var>proxies</var> be « "<code>DIRECT</code>" ».
+
+  <p class=note>This is where non-standard technology such as
+  <a href="https://en.wikipedia.org/wiki/Web_Proxy_Auto-Discovery_Protocol">Web Proxy Auto-Discovery Protocol (WPAD)</a>
+  and <a href="https://en.wikipedia.org/wiki/Proxy_auto-config">proxy auto-config (PAC)</a> come
+  into play.
```

Comment:
I don't know if it's a good thing to mention WPAD here; it's widely considered to be a huge security issue.

Reply:
Yes, but everyone implements it. Ignoring it won't make it go away.

Reply:
I would be open to pointing out the security issues.